src/isar_exr/api/energy_robotics_api.py

```diff
@@ -39,3 +42,26 @@ def is_mission_running(self, exr_robot_id: str):
         response_dict: dict[str, Any] = self.client.query(query_string, params)
         is_running: bool = response_dict["isMissionRunning"]
         return is_running
+
+    def pause_current_mission(self, exr_robot_id: str) -> None:
+        query_string: str = """
+            mutation pauseMission($robot_id: String!)
+                {
+                    pauseMissionExecution(robotID:$robot_id)
```

Comment:
Can you test that this function raises and returns as expected?

Reply:
Agree fixed